.gitattributes

```diff
@@ -19,3 +19,4 @@
 *.ttf               binary
 *.woff              binary
 *.woff2             binary
+*.xlsm              binary
```

Comment:
This prevents git from treating the test data file as text & "helpfully" correcting line endings